src/test/java/com/gavinfenton/quizolation/service/TeamServiceTest.java

```diff
@@ -0,0 +1,115 @@
+package com.gavinfenton.quizolation.service;
```

Comment:
I would give more descriptive names of what is being tested as testCreateTeam() etc isn't descriptive, if functionality was added to it then it could be assumed that this is covered when it isn't

I think names like testCreateTeamSavesToAndReturnsFromRepository() lets you know that this is the only thing being tested (though it's a bit long)